tests/test_qartod.py

```diff
@@ -622,19 +634,12 @@ def test_flat_line_missing_values(self):
             result = qartod.flat_line_test(
                 inp=i,
                 tinp=self.times,
-                thresholds=self.thresholds,
+                suspect_threshold=self.suspect_threshold,
+                fail_threshold=self.fail_threshold,
                 tolerance=self.tolerance
             )
             npt.assert_array_equal(result, expected)
 
-    def test_flat_line_bad_input(self):
```

Comment:
Why did you need to remove this?

Reply:
The flat line thresholds are seconds instead of counts, so I don't think this is a valid test case anymore.